README.md

```diff
@@ -194,11 +194,14 @@ and is replaced it with native CSS.
 2. Replace `@emotion/react` with `jotai-devtools/styles.css` in your code
 
 Note that this css file may get included in your production builds please import
-it conditionally if you want to avoid that.
+it conditionally if you want to avoid that, e.g.:
 
 ```diff
 import { DevTools } from 'jotai-devtools';
-+ import 'jotai-devtools/styles.css';
++ 
++ if (process.env.NODE_ENV === 'development') {
++     require('jotai-devtools/styles.css')
```

Comment:
Thanks for the PR! I'm not sure if I'd recommend using `require` due to its limited support 🤔 

What about lazy loading the component like this 👇? Would that for Next.Js?

```ts
const LazyDevtools = lazy(() => import('./JotaiDevTools').then((module) => ({ default: module.Devtools })));

export const YourApp = () => {
  return (
    <Suspense fallback="">
      { process.env.NODE_ENV === "development" ?  <LazyDevtools/> : null }
    </Suspense>
  );
}
```
In `./JotaiDevTools.tsx` 
```ts
import "jotai-devtools/styles.css";
export { DevTools } from "jotai-devtools";
```
---

**Vite users could do something like this 👇 **
```tsx
import { DevTools } from "jotai-devtools";
import css from "jotai-devtools/styles.css?inline";

export const JotaiDevTools = () =>
  process.env.NODE_ENV !== "production" ? (
    <>
      <style>{css}</style>
      <DevTools />
    </>
  ) : null;
  ```

Reply:
> However, for some reason when I do this the build file turns out to be MUCH larger.

@npearson72 - Thanks for reporting. That seems strange! 🤔 Works fine on my end (see screenshot below). Could you create a small repro? 

![Group 1854](https://github.com/jotaijs/jotai-devtools/assets/14841132/8dc37cb6-a754-4f18-8223-e84d4d9f89bf)

Reply:
@arjunvegda Here's what I'm seeing:

![Screenshot 2024-05-09 at 7 51 53 PM](https://github.com/jotaijs/jotai-devtools/assets/1323414/a575d791-599c-4464-8bc8-0804a93ebb60)

Repo: https://github.com/npearson72/jotai-devtools-test

Reply:
@arjunvegda have you looked at `jotai-devtools/dist/index.css`?

It has all the mantine styles in it. I'm assuming you wanted those to live in `jotai-devtools/dist/internal__devtools.css`, but perhaps I've misunderstood.

In any case, it seems v0.8.0, which is using Mantine 6x which is not having this issue. But all later versions do.

Reply:
Ah! you're missing `process.env.NODE_ENV !== "production"` check in [DevTools.tsx#L4](https://github.com/npearson72/jotai-devtools-test/blob/master/src/DevTools.tsx#L4). 

See the full code in the [above comment](https://github.com/jotaijs/jotai-devtools/pull/142#discussion_r1593243980).

Reply:
Oh, I see. I didn't realize that `process.env` was available in Vite, but that make sense. Thank you.